source/setup.Rmd

```diff
@@ -1,282 +1,277 @@
+are stored. In the "Container path" text box, enter `/home/jovyan/work`. Then click the "Run" button to start the container.
 
-```
-bash path/to/Miniconda3-latest-MacOSX-arm64.sh
+```{r docker-desktop-runconfig, echo = FALSE, fig.cap = "The Docker Desktop container run configuration menu.", fig.retina = 2, out.width="100%"}
+image_read("img/setup/docker-3.png") |> 
+  image_crop("3632x2000")
 ```
 
-Make sure to replace `path/to/` with the path of the folder
-containing the downloaded script. Most computers will save downloaded files to the `Downloads` folder. 
-If this is the case for your computer, you can run the script in the terminal by typing:
+After clicking the "Run" button, you will see a terminal. The terminal will then print 
+some text as the Docker container starts. Once the text stops scrolling, find the 
+URL in the terminal that starts 
+with `http://127.0.0.1:8888` (highlighted by the red box in Figure \@ref(fig:ubuntu-docker-terminal)), and paste it 
+into your browser to start JupyterLab.
 
-```
-bash Downloads/Miniconda3-latest-MacOSX-arm64.sh
+```{r docker-desktop-url, echo = FALSE, fig.cap = "The terminal text after running the Docker container in Ubuntu. The red box indicates the URL that you should paste into your browser to open JupyterLab.", fig.retina = 2, out.width="100%"}
+image_read("img/setup/docker-4.png") |> 
+  image_crop("3632x2000")
 ```
 
-The instructions for the installation will then appear. 
-Follow the prompts and agree to accepting the license,
-the default installation location,
-and to running `conda init`, which makes `conda` available from the terminal.
+## Working with JupyterLab Desktop
 
-**Ubuntu:** To install miniconda on Ubuntu, first download
-the [latest Python 64-bit version from here](https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh). 
-After the download has finished, open the terminal and execute the following 
-command:
+You can also run the worksheets accompanying this book on your computer
+using [JupyterLab Desktop](https://github.com/jupyterlab/jupyterlab-desktop).
+The advantage of JupyterLab Desktop over Docker is that it can be easier to install;
+Docker can sometimes run into some fairly technical issues (especially on Windows computers) 
+that require expert troubleshooting. The downside of JupyterLab Desktop is that there is a (very) small chance that 
+you may not end up with the right versions of all the R packages needed for the worksheets. Docker, on the other hand,
+*guarantees* that the worksheets will work exactly as intended. 
 
-```
-bash path/to/Miniconda3-latest-Linux-x86_64.sh
-```
+In this section, we will cover how to install Git (for version control, as discussed in Chapter \@ref(Getting-started-with-version-control)),
+JupyterLab Desktop, the JupyterLab Git extension, and all of the R packages needed to run the code in this book.
```

Comment:
I suggest this change because in the paragraph above we are focused on the JupyterLab software, and then we make a hard transition to saying we are going to install Git which we have not yet talked about in this chapter...
```suggestion
In this section, we will cover how to install JupyterLab Desktop, Git (for version control, as discussed in Chapter \@ref(Getting-started-with-version-control)), the JupyterLab Git extension, and all of the R packages needed to run the code in this book.
```

Reply:
I like this change. I'll give the original and your proposed version a read as I go through the chapter + your comments

Reply:
I reworded slightly to just put JLab Desktop first